documentation/docs/scenarios/upload-to-transportrequest/Readme.md

```diff
@@ -29,7 +29,7 @@ This scenario combines various different steps to create a complete pipeline.
 
 In this scenario, we want to show how to build an application based on SAPUI5 or SAP Fiori by using the multitarget application (MTA) concept and how to attach the build result to a transport request inside an ABAP system. This document comprises the [mtaBuild](../../../steps/mtaBuild/) and the [transportRequestUploadFile](../../../steps/transportRequestUploadFile/) steps.
 
-In case of an RFC based upload the binary is not streamed to the ABAP endpoint. Instead an URL pointing to the binary needs to be provided. Hence the binary must be published first so that it can be accessed via HTTP. This can happen by uploading the binary to a blob store or by archiving the artifact on Jenkins. The corresponding URL needs to be provided when the artifact is attached to the transport request.
+For the RFC based upload the binary is not streamed to the ABAP endpoint. Instead an URL pointing to the binary needs to be provided. Hence the binary must be published first so that it can be accessed via HTTP. This can happen by uploading the binary to a blob store or by archiving the artifact on Jenkins. The corresponding URL needs to be provided when the artifact is attached to the transport request.
```

Comment:
```suggestion
For the RFC based upload, the binary is not streamed to the ABAP endpoint. Instead, an URL pointing to the binary needs to be provided. To publish the binary first so that it can be accessed via HTTP, upload it to a blob store or archive the artifact on Jenkins. The corresponding URL needs to be provided when the artifact is attached to the transport request.
```